pkg/controller/dataimportcron-controller.go

```diff
@@ -248,15 +249,21 @@ func splitImageStreamName(imageStreamName string) (string, string, error) {
 	return "", "", errors.Errorf("Illegal ImageStream name %s", imageStreamName)
 }
 
-func (r *DataImportCronReconciler) pollImageStreamDigest(ctx context.Context, dataImportCron *cdiv1.DataImportCron) (reconcile.Result, error) {
+func (r *DataImportCronReconciler) pollSourceDigest(ctx context.Context, dataImportCron *cdiv1.DataImportCron) (reconcile.Result, error) {
 	if nextTimeStr := dataImportCron.Annotations[AnnNextCronTime]; nextTimeStr != "" {
```

Comment:
nit: return here and remove all function indentation

Reply:
I see you are in code quality mood. This is dusty code:)

Reply:
I know but if you are in the neighborhood you can beautify the houses lol